demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1023,3 +1025,15 @@ def get_packs_that_should_have_version_raised(modified_files, changed_meta_packs
         })
 
         return changed_meta_packs.union(modified_packs_that_should_have_version_raised)
+
+    @staticmethod
+    def get_packs(changed_files):
+        packs = set()
+        for changed_file in changed_files:
+            if isinstance(changed_file, tuple):
+                changed_file = changed_file[1]
+            pack = get_pack_name(changed_file)
+            if pack and is_file_path_in_pack(changed_file):
```

Comment:
why do you need to call `is_file_path_in_pack`?
if the file is not in a pack, `pack` will be None.